docs/getting-started/integrations-cdn/laravel.md

```diff
@@ -18,6 +18,10 @@ This guide assume you have a Laravel project. You can create a basic Laravel pro
 
 ## Using from CDN
 
+<info-box>
+	To use our CDN services, you need [a free account](https://portal.ckeditor.com/checkout?plan=free).
```

Comment:
I'm assuming the link will work after releasing the portal?

Reply:
yes, you can check the flow with ckeditor-dev.com